dali/operators/decoder/cache/cached_decoder_impl.cc

```diff
@@ -39,7 +39,7 @@ CachedDecoderImpl::CachedDecoderImpl(const OpSpec& spec)
         device_id_, cache_type, cache_size, cache_debug, cache_threshold);
 
       use_batch_copy_kernel_ = spec.GetArgument<bool>("cache_batch_copy");
-      auto batch_size = spec.GetArgument<int>("batch_size");
+      auto batch_size = spec.GetArgument<int>("max_batch_size");
```

Comment:
Now that's a bug, isn't it?

Reply:
As I wrote in the description, I'm not reworking decoders and random operators in this PR.